modules/gcloud/src/main/java/org/testcontainers/containers/DatastoreEmulatorContainer.java

```diff
@@ -21,13 +21,32 @@ public class DatastoreEmulatorContainer extends GenericContainer<DatastoreEmulat
 
     private static final int HTTP_PORT = 8081;
 
+    private String flags;
+
+    public DatastoreEmulatorContainer(final String image) {
+        this(DockerImageName.parse(image));
```

Comment:
WDYT of adding `.withFlags("--consistency 1.0")` here, as a good default for testing, as mentioned in the original issue?

Reply:
given not a datastore expert I would like to leave it open due to

> At most one of these can be specified:
--consistency=CONSISTENCY; default=0.9
Fraction of eventually consistent operations that should succeed immediately. Setting to 1.0 can be useful for unit tests, but may mask incorrect assumptions about non-ancestor queries which are eventually consistent.
--use-firestore-in-datastore-mode
Runs the emulator in Cloud Firestore in Datastore Mode. Reads are always strongly consistent and --consistency flag may not be specified.

According to the docs one should be specified, so, users can make this decision.

Reply:
Agreed, since we are not the expert, let's leave the default unless an expert with a stronger opinion suggests otherwise 👍